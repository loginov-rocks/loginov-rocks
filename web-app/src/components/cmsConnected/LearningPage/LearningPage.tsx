import { Link } from '@reach/router';
import * as React from 'react';

import { CmsComponent } from 'cms/interfaces/CmsComponent';
import { CmsConnectedProps } from 'cms/interfaces/CmsConnectedProps';

interface Props extends CmsConnectedProps {
  sections: CmsComponent[];
  title: string;
}

export const LearningPage: React.FC<Props> = ({ render, sections, title }) => (
  <>
    <h1>{title}</h1>
    <nav>
      <Link to="/">Home</Link>
      {' '}
      /
      {' '}
      Learning
    </nav>
    {render(sections)}
  </>
);
