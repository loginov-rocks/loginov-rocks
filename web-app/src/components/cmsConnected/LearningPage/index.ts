import { LearningPage } from './LearningPage';

export { LearningPage };
