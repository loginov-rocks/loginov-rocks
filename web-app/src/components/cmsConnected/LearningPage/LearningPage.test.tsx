/* eslint-disable import/no-extraneous-dependencies */

import { shallow } from 'enzyme';
import * as React from 'react';

import { LearningPage } from './LearningPage';

it('matches snapshot', () => {
  const wrapper = shallow(<LearningPage render={() => null} sections={[]} title="Learning" />);

  expect(wrapper).toMatchSnapshot();
});
