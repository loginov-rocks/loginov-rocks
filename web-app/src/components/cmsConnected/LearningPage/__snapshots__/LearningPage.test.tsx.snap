// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Fragment>
  <h1>
    Learning
  </h1>
  <nav>
    <Link
      to="/"
    >
      Home
    </Link>
     
    /
     
    Learning
  </nav>
</Fragment>
`;
